app/src/main/java/com/google/ytdl/UploadsListFragment.java

```diff
@@ -46,23 +49,35 @@
  *         Left side fragment showing user's uploaded YouTube videos.
  */
 public class UploadsListFragment extends Fragment implements ConnectionCallbacks,
-        OnConnectionFailedListener {
+        OnConnectionFailedListener, GoogleApiClient.ConnectionCallbacks, GoogleApiClient.OnConnectionFailedListener {
```

Comment:
Adding a few more listeners and callbacks so that we can use the GoogleapiClient.